Library/Homebrew/cask/lib/hbc/cli/internal_appcast_checkpoint.rb

```diff
@@ -2,13 +2,14 @@ module Hbc
   class CLI
     class InternalAppcastCheckpoint < InternalUseBase
       def self.run(*args)
-        cask_tokens = cask_tokens_from(args)
+        calculate = args.include? "--calculate"
+        cask_tokens = args.reject { |arg| arg.chars.first == "-" }
```

Comment:
It was overlooked. It does indeed work, but I was basing myself on `_stanza` and didn’t even notice. Will fix it and add to https://github.com/Homebrew/brew/pull/1896 as well.